src/main/groovy/edu/wpi/first/gradlerio/wpi/dependencies/WPIVendorDepsExtension.groovy

```diff
@@ -45,6 +57,55 @@ public class WPIVendorDepsExtension {
             return []
     }
 
+    void loadAll() {
+        loadFrom(vendorFolder)
+    }
+
+    void loadFrom(File directory) {
+        vendorFiles(directory).each { File f ->
+            def dep = parse(f)
+            if (dep != null)
+                load(dep)
+        }
+    }
+
+    void loadFrom(Project project) {
+        loadFrom(project.file(VENDORDEPS_FOLDER_NAME))
+    }
+
+    JsonDependency parse(File f) {
+        JsonDependency dep = null
+        f.withReader {
+            try {
+                dep = parseSlurped(slurper.parse(it))
+            } catch (ex) {
+                log.logError("Malformed Vendor Deps File: ${f.toString()}")
```

Comment:
We don't want to be printing the stacktrace here, no. Java stacktraces are too long and useless and it would make it impossible to actually find any other errors.

Reply:
The stacktrace doesn't include any information about the malformed JSON, just a call stack, so it's pretty useless in this case.